src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy

```diff
@@ -266,7 +266,8 @@ class IslReplugSpec extends HealthCheckSpecification {
         database.resetCosts(topology.isls)
     }
 
-    @Ignore("https://github.com/telstra/open-kilda/issues/3780")
+//    @Ignore("https://github.com/telstra/open-kilda/issues/3780")
+//    test need supervision (on hardware environment) cause vm is not match to the test requirements
```

Comment:
why comment? not delete?

Reply:
left it due to the test is not tested on hardware env, so to take supervision after
but you are right, delete it